### PR TITLE
Iosxe/show cts role based counters

### PR DESCRIPTION
**Pull request by reustaquio22**
Parser for Iosxe/show cts role based counters

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_cts.py [hunk 1]
```diff
@@ -149,3 +148,94 @@ def filter_lines(raw_output, remove_lines):
...
+# Schema for:
+#  * 'show cts role-based counters'
+# =================================
+class ShowCtsRoleBasedCountersSchema(MetaParser):
+    """Schema for show cts role-based counters."""
+
+    schema = {
+        "cts_rb_count": {
+            int: {
+                "src_group": Or(str, int),
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/show_cts.py:163]: I would recommend to use only `str` for `src_group` and `dst_group` if the value could be both. please change to just `str`.

other than this, schema looks good

**Comment by ahersi2-cisco** [comment]
Looks good, if you just address the two comments, I'll go ahead and approve it, thanks!

File: src/genie/libs/parser/iosxe/show_cts.py (hunk 1 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_cts.py]: This has been updated

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py [hunk 2]
```diff
@@ -52,5 +52,149 @@ def test_show_cts_pacs_empty(self):
...
+                "sw_permit_count": 0,
+                "hw_permit_count": 0,
+                "sw_monitor_count": 0,
+                "hw_monitor_count": 0
+            }
+        }
+    }
+
+    golden_output1 = {'execute.return_value': '''
+Role-based IPv4 counters
```
[the remark this answers is not in the document]
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/tests/test_show_cts.py]: This has been updated.

**Comment by ahersi2-cisco** [comment]
One more merge conflict, and I can merge right away, thanks!

**Comment by reustaquio22** [comment]
@tahigash, conflicts were resolved.

**Comment by ahersi2-cisco** [comment]
Hi, can you please resolve the conflicts for the changelog, I can merge it right after thats handled, thanks.

File: src/genie/libs/parser/iosxe/show_cts.py (hunk 1 above)
Comment (tahigash) [reply src/genie/libs/parser/iosxe/show_cts.py]: confirmed

**Review by tahigash — APPROVED** [review]

**Review by ahersi2-cisco — APPROVED** [review]

**Pull request merged**